packages/test/test-service-load/src/gcDataStores.ts

```diff
@@ -197,26 +194,37 @@ export class DataObjectNonCollab extends BaseDataObject implements IGCDataStore
     public async run(config: IRunConfig, id?: string): Promise<boolean> {
         console.log(`########## Started child [${id}]`);
         assert(config.testConfig.inactiveTimeoutMs !== undefined, "inactive timeout is required for GC tests");
+
+        this.myId = id;
+        this.shouldRun = true;
         // Set the local inactive timeout 500 less than the actual to keep buffer when marking data stores as inactive.
         this._inactiveTimeoutMs = config.testConfig.inactiveTimeoutMs - 500;
-        // Set up the child to send ops opRateMultiplierPerLevel times faster than this data store.
-        const opRatePerMin = config.testConfig.opRatePerMin * opRateMultiplierPerLevel;
+
+        /**
+         * Adjust the totalSendCount and opRatePerMin. There can be maximum of maxRunningLeafChildren children running
+         * at the same time.
+         * - This data store and each child sends 1/4th of the number of ops per min.
+         * - This data store sends 1/4th the number of total ops since its sending at 1/4th the op rate.
+         */
+        const opRatePerMin = Math.ceil(config.testConfig.opRatePerMin / (maxRunningLeafChildren + 1));
+        const totalSendCount = config.testConfig.totalSendCount / 4;
         this._childRunConfig = {
             ...config,
             testConfig: {
                 ...config.testConfig,
                 opRatePerMin,
+                totalSendCount,
             },
         };
-        this.myId = id;
-        this.shouldRun = true;
 
-        const delayBetweenOpsMs = 60 * 1000 / config.testConfig.opRatePerMin;
+        // Perform an activity every 1/6th minute = every 10 seconds.
+        const activityThresholdOpCount = Math.ceil((opRatePerMin / 6));
         let localSendCount = 0;
```

Comment:
I'm worried about how localSendCount v. totalSendCount are used. Some observations:

* totalSendCount assumes config value is shared across all self/child data stores and all client collaborating on those data stores
* from nodeStressTest.ts: `estRunningTimeMin = Math.floor(profile.totalSendCount / profile.opRatePerMin);` so totalSendCount and opRatePerMin are tied together.  Meaning, opRatePerMin is across all clients.
* But delayBetweenOpsMs and activityThresholdOpCount is tied to opRatePerMin and localSendCount... that seems wrong?  You accounted for data store tree (div by 4) but not multiple clients collaborating.

Reply:
The opRatePerMin is adjusted by the RootDataStore as per the number of clients. The totalSendCount is not adjusted based on number of clients because it is shared among all clients via the shared counter.
The RootDataStore also divides the totalSendCount and opRatePerMin across the 2 children so that each is responsible for just its own counter / op-rate.
The basic idea I had in mind was that each object in the hierarchy is responsible for controlling the behavior of itself and its children.

Reply:
I see the following issue - The total number of ops in the test will most likely be less than totalSendCount. This is because the collab / non-colalb data stores only send 1/maxDataStores number of ops and rely on the children to send rest of the ops. There may be less than max children and they may be sending ops at a lower or higher frequency.

We could add logic to count ops sent by all the children of the collab / non-collab data store and take them into account. But, do we really need that complication? It's not perfect currently but is it good enough?